File: SignalRBackend.DAL/DomainModels/Message.cs
using System;

namespace SignalRBackend.DAL.DomainModels
{
    public class Message
    {
        public Int32? Id { get; set; }
        public Int32 ChatId { get; set; }
        public Int32 UserId { get; set; }
        public Int32? ReceiverId { get; set; }
        public String UserName { get; set; }
        public  Int32? RepliedMessageId { get; set; }
        public Boolean IsDeletedOnlyForCreator { get; set; }
        public String MessageText { get; set; }
        public DateTime? ActivityDate { get; set; }
        public Message RepliedMessage { get; set; }
        public User Receiver { get; set; }
        public User User { get; set; }
        public Chat Chat { get; set; }

    }
}

File: SignalRFrontend/src/models/message.ts
import { Chat } from "./chat";
import { User } from "./user";

export class Message{
    id: number | null = null;
    chatId: number = 1;
    userId: number = 1;
    receiverId: number | null = null;
    userName: string | null = "";
    messageText: string  = "";
    isDeletedForMe: boolean = false;
    activityDate: Date | null = null;
    user: User | null= null;
    receiver: User | null= null;
    chat: Chat | null= null;
}